src/components/inputs/rich-text-input/README.md

```diff
@@ -0,0 +1,31 @@
+# RichTextInput
+
+#### Description
+
+A controlled rich text input component for rich text with validation
+states.
+
+This component uses `slatejs` under the hood. This means that the `value` needs to be kept in slate format.
+The `RichTextInput` exposes two static helper functions to helper with transforming HTML values to and from slate format.
+
+`RichTextInput.deserialize(html)` can be used to turn HTML into slate format.
```

Comment:
Why do we need to expose this two? Why not let end-user always work with `html` and do de-/serialization inside this component?

Reply:
Basically the performance of slate becomes unusable if we are constantly serializing / deserializing. Plus, then we lose all the undo / redo state.